### PR TITLE
remove deprecated event initializer

### PR DESCRIPTION
**Pull request by ndurell**
# Description

Removes access to the event initializer (still used internally).

# Check List

- [ ] Are you changing anything with the public API?
- [ ] Have you tested this change on real device?
- [ ] Are your changes backwards compatible with previous SDK Versions?
- [ ] Have you added unit test coverage for your changes?
- [ ] Have you verified that your changes are compatible with all the operating system version this SDK currently supports?

# Manual Test Plan

1.

# Supporting Materials

### CODE REVIEW

File: Sources/KlaviyoSwift/KlaviyoModels.swift [hunk 1]
```diff
@@ -26,7 +26,7 @@ public struct Event: Equatable {
         }
     }
 
-    public struct Identifiers: Equatable {
+    struct Identifiers: Equatable {
```
Comment (ndurell) [inline Sources/KlaviyoSwift/KlaviyoModels.swift:29]: These should have been deprecated...

File: Sources/KlaviyoSwift/StateManagement.swift [hunk 2]
```diff
@@ -508,23 +508,10 @@ extension KlaviyoState {
 
 extension Event {
     func updateStateAndEvent(state: inout KlaviyoState) -> Event {
-        let email = identifiers?.email ?? state.email
-        let phoneNumber = identifiers?.phoneNumber ?? state.phoneNumber
-        let externalId = identifiers?.externalId ?? state.externalId
         let identifiers = Identifiers(
```
Comment (ndurell) [inline Sources/KlaviyoSwift/StateManagement.swift:511]: This no longer updates the state.

File: Sources/KlaviyoSwift/KlaviyoModels.swift (hunk 1 above)
Comment (evan-masseau) [reply Sources/KlaviyoSwift/KlaviyoModels.swift:29]: I suppose you could still release a 2.6 where you deprecate it. Kinda pedantic.

**Review by evan-masseau — APPROVED** [review]

File: Sources/KlaviyoSwift/KlaviyoModels.swift (hunk 1 above)
Comment (ndurell) [reply Sources/KlaviyoSwift/KlaviyoModels.swift:29]: That was my thought too. I don't think you would use one without the other.

**Pull request merged**